modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceAccountRoleHelperImpl.java

```diff
@@ -308,9 +308,6 @@ else if (name.equals(AccountRoleConstants.ROLE_NAME_SUPPLIER)) {
 				new String[] {
 					CommercePriceListActionKeys.ADD_COMMERCE_PRICE_LIST
 				});
-			companyResourceActionIds.put(
```

Comment:
Why are we reverting this permission? Isn't the supplier supposed to access price lists and promotions if he's linked to a catalog/channel?

Reply:
so this need to be reverted because with this permission set the prefilter made by the search engine is not working and causes the bug https://liferay.atlassian.net/browse/COMMERCE-13195

Reply:
Shouldn't we working on the pre-filter then? I really don't get how removing a permission is giving the supplier access to the resource